src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,139 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl service;
+    private static User user1;
+    private static User user2;
+    private static User user3;
+    private static final long DEFAULT_ID = 200L;
+    private static final String DEFAULT_LOGIN = "sasaylalita";
+    private static final String DEFAULT_PASSWORD = "qwerty12345";
+    private static final int DEFAULT_AGE = 28;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user1 = new User();
+        user1.setId(DEFAULT_ID);
+        user1.setLogin(DEFAULT_LOGIN);
+        user1.setPassword(DEFAULT_PASSWORD);
+        user1.setAge(DEFAULT_AGE);
+        user2 = new User();
+        user2.setId(DEFAULT_ID);
+        user2.setLogin(DEFAULT_LOGIN);
+        user2.setPassword(DEFAULT_PASSWORD);
+        user2.setAge(DEFAULT_AGE);
+        user3 = new User();
+        user3.setId(DEFAULT_ID);
+        user3.setLogin(DEFAULT_LOGIN);
+        user3.setPassword(DEFAULT_PASSWORD);
+        user3.setAge(DEFAULT_AGE);
+    }
+
+    @Test
+    void registration_validData_ok() {
+        User expected = user1;
+        service.register(expected);
+        User actual = Storage.people.get(0);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void isValid_hasSixChar_ok() {
```

Comment:
```suggestion
    void isValid_hasSixChar_ok() {
```
From this naming, I do not understand what is tested. Let's clarify that it is login.

Also, what would be if requirement changes and we checking not 6, but 8 chars?
Let's not be connected to it in our naming. 

Review all your naming with such patterns.